libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -97,7 +97,7 @@ TODO
 
 ABI Affecting Changes
 ---------------------
-TODO
+ - LWG3430: Disallow implicit conversion of the source arguments to ``std::filesystem::path`` when constructing ``std::basic_*fstream``.
```

Comment:
I think this is better placed under `Deprecations and Removals`. 

This text is technically correct. However the target audience of the release notes are typical developers and not people who read LWG issues and papers. So let's improve this text by explaining what it does. For example, 
```
This effectively removes the possibility to directly construct a ``std::basic_fstream`` from a
``std::basic_stringview`` or a C-string, instead you can construct a temporary ``std::basic_string``.
This change has been applied to C++17 and later.
```
Feel free to write this in your own words.

Reply:
Thanks, I've update release notes wording.